apps/concierge_site/lib/views/route_helper.ex

```diff
@@ -30,13 +32,12 @@ defmodule ConciergeSite.RouteHelper do
   def route_name("Green-E"), do: "Green Line E"
 
   def route_name(route_id) do
-    {:ok, route} = ServiceInfoCache.get_route(route_id)
-
-    # We never want long names for buses
-    if route.route_type == 3 || route.long_name == "" do
-      Route.bus_short_name(route)
-    else
-      route.long_name
+    case ServiceInfoCache.get_route(route_id) do
+      # We never want long names for buses
+      {:ok, %{route_type: 3} = route} -> Route.bus_short_name(route)
+      {:ok, %{long_name: ""} = route} -> Route.bus_short_name(route)
```

Comment:
Codecov is calling out this line as not being covered by tests. Is it easy to add a case for this? I realize it's existing functionality not related to your changes, so it's fine if it's not trivial or doesn't seem like the time and place.

Reply:
On closer inspection it seems like this was sort of duplicating logic that already existed in the form of `Route.name`, so I'm replacing it with a call to that. It's not exactly the same, but it passes all the current tests and I'm honestly not sure under what circumstance we'd want the "`bus_short_name`" behavior for routes that are not bus routes.